src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -1,51 +1,73 @@
 package com.volunteer.enterprise;
-import dto.Volunteer;
+import com.volunteer.enterprise.dto.Volunteer;
+import com.volunteer.enterprise.service.IVolunteerService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
 
+import java.util.List;
+
 @Controller
 public class VolunteerAppController {
+    IVolunteerService volunteerService;
 
- @Autowired
- VolunteerService volunteerService;
-
+    /**
+     *RequestMapping for root (/) endpoint
+     * Create a Voluntier volunteer object then display start
+     * @return Voluntier index page
+     */
 
     @RequestMapping("/")
     public String index(Model model){
 
-Volunteer volunteer = new Volunteer();
-
-volunteer.setVolunteerID(1);
-volunteer.setVolunteerUser("Iron Man");
-volunteer.setVolunteerPass("1234");
+    Volunteer volunteer = new Volunteer();
+    volunteer.setVolunteerID(1);
+    volunteer.setVolunteerUser("Iron Man");
+    volunteer.setVolunteerPass("1234");
+    model.addAttribute(volunteer);
 
-model.addAttribute(volunteer);
-        return "Index";
+        // Add newly created favorite to model so it can be displayed
+        model.addAttribute(volunteer);
+        return "start";
 
     }
 
-
+    /**
+     * RequestMapping for /saveVolunteer endpoint
+     * Save a nww volunteer with details provided via HTTP query string
+     * @param volunteer provided through HTTP query
+     * @return Volunteer Index page displaying newly saved volunteer
+     */
 
 
 
     @RequestMapping("/saveVolunteer")
-    public String saveVolunteer(Volunteer volunteer){
-
-        return "Index";
+    public String saveVolunteer(Volunteer volunteer, Model model){
+        try {
+            volunteerService.save(volunteer);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
If you introduce a try-catch block, make use of the catch block.  No one will see the print stack trace, and the program will move on to the next line (model.addAttribute()) in all cases: if the try is successful or not.  Consider why you get to the exception block (because of an error), and how you'd handle that error.